### DISCUSSION TITLE
Save, new and discard buttons in the editors

### DISCUSSION THREAD

**Discussion by mikfire** [Brewtarget/brewtarget#574] [General]
Some of the editors (yeast, for example) show two buttons labeled "OK" (to save your edits) and "Cancel" (to discard your edits). Other editors (equipment, for example) show a "+", a floppy disk and a red power button to create a new profile, save your changes to the current profile or discard them.

I would like to make them all consistent. My initial playing around actually suggest using both icons and words actually works. 

This will be a non-trivial effort, and so I am wondering if anybody has a strong opinion before I do this work.

The choice, to be clear, is to use icons, words or maybe both. The choice is not if they will be made consistent so that all editors have the same three actions, because that is going to happen.

**Comment by mattiasmaahl** [comment]
I have no strong Opinion, can have it either way but leaning more against using words.

**Reply by matty0ung** [reply]
Sounds good to make everything consistent.  If we go with icons, I'd also vote for ensuring we have tooltips.  (Some icons are memorable once you know them but cryptic at first - eg the raincoat we use for "style"!  And probably lots of old people like me know what a floppy disk is and guess it means save, but I don't know if that's true for folks born this century etc. :->)

**Reply by mikfire** [reply]
That's supposed to be a tuxedo jacket. Between that and a floppy disk, you see my problem. And I suspect the floppy icon will mean save for a long time after people forget why it means that.

Since none of us have strong opinions, I will go with the icons and good tool tips.
